fe1-web/src/features/meeting/__tests__/utils.ts

```diff
@@ -4,7 +4,7 @@ import { Timestamp } from 'core/objects';
 import { CreateMeeting } from '../network/messages';
 import { Meeting, MeetingState } from '../objects';
 
-const TIMESTAMP = new Timestamp(1609455600); // 1st january 2021
+const TIMESTAMP = new Timestamp(1610000000); // 7th of january 2021
```

Comment:
It does but unfortunately it is not 😂 The timestamp 1609455600 (since it is the 1st of january) results in different years depending on the timezone of the machine.. (i.e. my machine and the CI disagreed)

Hence I moved it further into the future 😂 Guess we should avoid the 1st of january

Reply:
Wow, I didn't expect such an elaborate reasoning behind this change.
I just thought you wanted a round number 🤣